genesis-io/nexus_instinct
# Feat/nxi 3

#### Relevant Issues
https://github.com/genesis-io/nexus_mineral/issues/5
#### Description of PR
Create landing page for application
#### Manual test procedure
checkout branch feat/NXI-3
*yarn install 
*yarn start
*go to browser localhost:3000
*verify that there is a landing page on home route
